docs/model.rst

```diff
@@ -36,7 +36,7 @@ and even perform operations on multiple records (See `Persistence Actions` below
 
    $m->action('delete')->execute(); // performs mass delete, hooks are not executed
    
-   $m->each('delete'); // deletes each record, hooks are executed
+   $m->each(function () use ($m) { $m->delete(); }); // deletes each record, hooks are executed
```

Comment:
Can you change the $m in docs to $model or even $invoice or whatever it refers to?